src/Compilers/CSharp/Portable/Binder/Binder_Operators.cs

```diff
@@ -435,6 +435,18 @@ private BoundExpression BindSimpleBinaryOperator(BinaryExpressionSyntax node, Di
 
             if (left.HasAnyErrors || right.HasAnyErrors)
             {
+                if (node.OperatorToken.IsKind(SyntaxKind.MinusToken)
```

Comment:
>if (node.OperatorToken.IsKind(SyntaxKind.MinusToken) [](start = 16, length = 52)

This doesn't feel like the best place to report this error. See my comment in the caller function above.